Module4_Labs/Lab2_Doubly_Linked_List/Cards/2.md

```diff
@@ -8,4 +8,6 @@
 
 You can traverse the list to get each element of the node and do something with it, either printing each element in the list or comparing it with a specific value.
 
-Under your doubly linked list class, define a function call **printList()**. It should traverse the list and print each element separated by a comma.
\ No newline at end of file
+Under your doubly linked list class, define a function call **printList()**. It should traverse the list and print each element separated by a comma.
```

Comment:
add the parameters that printList() will require

I think the node parameter is the node that the list will start printing on. So it will be from a specified node to the end of the list.